style_exceptions/binary_bootstrap_formula_urls_allowlist.json

```diff
@@ -8,11 +8,6 @@
   "ghc@8.8",
   "ghc@9",
   "go",
-  "go-boring",
```

Comment:
Why `go-boring`?

Reply:
It's in the commit message.

> Also, remove `go-boring`, since it bootstraps using the Go formula and
not a binary artefact.

Reply:
Ha ha I never read commit messages.  Maybe I should :)